src/Microsoft.DotNet.Cli.Utils/IEnvironmentProvider.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using Microsoft.Extensions.PlatformAbstractions;
+
+namespace Microsoft.DotNet.Cli.Utils
+{
+    public interface IEnvironmentProvider
```

Comment:
Should we change the name of this interface. I don't correlate too much to the methods inside of it. But maybe I don't have enough context.